cms/navigation/tests/factories.py

```diff
@@ -0,0 +1,68 @@
+import factory
+from wagtail_factories import ListBlockFactory, PageChooserBlockFactory, StreamFieldFactory, StructBlockFactory
+
+from cms.core.tests.factories import LinkBlockFactory
+from cms.navigation.blocks import SectionBlock, ThemeLinkBlock, TopicLinkBlock
+from cms.navigation.models import ColumnBlock, MainMenu, NavigationSettings
+from cms.themes.tests.factories import ThemePageFactory
+from cms.topics.tests.factories import TopicPageFactory
+
+
+class ThemePageChooserFactory(PageChooserBlockFactory):
```

Comment:
Non-blocking: I've personally always hated block factories. Historically, I've always recommend that devs just get comfortable with raw block data representations by defining real life objects and inspecting `object.streamfield_name.raw_data` in the shell to see what they look like. It's basically just a list of dicts in a very predictable structure, which you can provide as a value to a new object.

Reply:
Let's follow this up.

Reply:
Sorry Andy, I've resolved to unblock merging.